internal/db/schema/migrations/oss/postgres/60/01_credential_vault_ssh_cert_library.up.sql

```diff
@@ -26,7 +26,7 @@ begin;
       check (
         name in (
           'ed25519',
-          'ecdsa',
+          'ec',
```

Comment:
we want to refer to this as `ecdsa`, so this shouldnt change. Instead you should translate `ecdsa` to `ec` when crafting the request body to vault.